src/cloudant/client.py

```diff
@@ -56,6 +55,13 @@ class CouchDB(dict):
     :param bool auto_renew: Keyword argument, if set to True performs
         automatic renewal of expired session authentication settings.
         Default is False.
+    :param int timeout: the read and connect timeout.  The connect timeout
+        is the number of seconds Request will wait for the client to establish a
+        connection to a Cloudant or a CouchDB account. The read timeout is the
+        number of seconds the client will wait for Cloudant or CouchDB to send
+        a response.  This timeout value will be applied to both the connect and
+        the read timeouts.  Specify a tuple if you would like to set the values
+        separately.  Default is 5 minutes.
```

Comment:
It may be best to leave it as `None` until the next version.  I'm under the assumption that more people would look at the CHANGES log for a major version and review for any behavior changes.